commit 2fb29226487e83979f2b674688f4c1127314986e
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Fri Nov 25 09:44:14 2022 -0800

    Most of Greenland stops clock-changing after March
    
    * NEWS: Mention this.
    * europe (America/Scoresbysund, America/Nuuk):
    Add an hour to UTC offset starting 2023-03-25 22:00 Nuuk time.
    No DST after that.

diff --git a/NEWS b/NEWS
index 24f77a74..9465c75e 100644
--- a/NEWS
+++ b/NEWS
@@ -10,6 +11,10 @@ News for the tz database

   Changes to future timestamps

+    Most of Greenland stops observing winter time after March 2023,
+    so its daylight saving time becomes standard time.  This affects
+    America/Nuuk and America/Scoresbysund.  (Thanks to Jürgen Appel.)
+
     Mexico will no longer observe DST after 2022, except for areas
     near the US border that continue to observe US DST rules.
     On 2022-10-30 at 02:00 the Mexican state of Chihuahua moves

diff --git a/europe b/europe
index 6ac41e72..d400a3ae 100644
--- a/europe
+++ b/europe
@@ -1103,7 +1103,19 @@ Zone Atlantic/Faroe	-0:27:04 -	LMT	1908 Jan 11 # Tórshavn
 # "National Park" by Executive Order:
 # http://naalakkersuisut.gl/~/media/Nanoq/Files/Attached%20Files/Engelske-tekster/Legislation/Executive%20Order%20National%20Park.rtf
 # It is their only National Park.
+
+# From Jürgen Appel (2022-11-25):
+# https://ina.gl/samlinger/oversigt-over-samlinger/samling/dagsordener/dagsorden.aspx?lang=da&day=24-11-2022
+# If I understand this correctly, from the next planned switch to
+# summer time, Greenland will permanently stay at that time, i.e. no
+# switch back to winter time in 2023 will occur.
 #
+# From Paul Eggert (2022-11-25):
+# For now, assume that this applies simultaneously to Ittoqqortoormiit
+# (currently -01) as well as to Nuuk (currently -03), and that
+# Danmarkshavn and Thule are unaffected since they are not using EU
+# rules anyway and some of the remarks were about EU rules.
+
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Thule	1991	1992	-	Mar	lastSun	2:00	1:00	D
 Rule	Thule	1991	1992	-	Sep	lastSun	2:00	0	S
@@ -1123,10 +1135,12 @@ Zone America/Danmarkshavn -1:14:40 -	LMT	1916 Jul 28
 Zone America/Scoresbysund -1:27:52 -	LMT	1916 Jul 28 # Ittoqqortoormiit
 			-2:00	-	-02	1980 Apr  6  2:00
 			-2:00	C-Eur	-02/-01	1981 Mar 29
-			-1:00	EU	-01/+00
+			-1:00	EU	-01/+00	2023 Mar 25 24:00
+			 0:00	-	+00
 Zone America/Nuuk	-3:26:56 -	LMT	1916 Jul 28 # Godthåb
 			-3:00	-	-03	1980 Apr  6  2:00
-			-3:00	EU	-03/-02
+			-3:00	EU	-03/-02	2023 Mar 25 22:00
+			-2:00	-	-02
 Zone America/Thule	-4:35:08 -	LMT	1916 Jul 28 # Pituffik
 			-4:00	Thule	A%sT
 
commit 36acd567fbd8df3e085351bec36974fde035e0f0
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Mon Nov 28 18:19:02 2022 -0800

    Greenland change should affect only America/Nuuk
    
    Although the situation is fluid, it seems I likely guessed wrong
    about Ittoqqortoormiit in my Greenland changes three days ago.
    (Thanks to Jonas Nyrup.)
    * NEWS: Mention this.
    * europe (America/Scoresbysund): Revert recent change.

diff --git a/NEWS b/NEWS
index 9465c75e..d819bf1b 100644
--- a/NEWS
+++ b/NEWS
@@ -13,9 +13,9 @@ Unreleased, experimental changes

   Changes to future timestamps
 
-    Most of Greenland stops observing winter time after March 2023,
-    so its daylight saving time becomes standard time.  This affects
-    America/Nuuk and America/Scoresbysund.  (Thanks to Jürgen Appel.)
+    Much of Greenland, represented by America/Nuuk, stops observing
+    winter time after March 2023, so its daylight saving time becomes
+    standard time.  (Thanks to Jonas Nyrup and Jürgen Appel.)

     Mexico will no longer observe DST after 2022, except for areas
     near the US border that continue to observe US DST rules.
diff --git a/europe b/europe
index d400a3ae..a3547b4f 100644
--- a/europe
+++ b/europe
@@ -1104,17 +1104,28 @@ Zone Atlantic/Faroe	-0:27:04 -	LMT	1908 Jan 11 # Tórshavn
 # http://naalakkersuisut.gl/~/media/Nanoq/Files/Attached%20Files/Engelske-tekster/Legislation/Executive%20Order%20National%20Park.rtf
 # It is their only National Park.
 
+# From Jonas Nyrup (2022-11-24):
+# On last Saturday in October 2023 when DST ends America/Nuuk will switch
+# from -03/-02 to -02/-01
+# https://sermitsiaq.ag/forslagtidsforskel-danmark-mindskes-sommertid-beholdes
+# ...
+# https://sermitsiaq.ag/groenland-skifte-tidszone-trods-bekymringer
+#
 # From Jürgen Appel (2022-11-25):
 # https://ina.gl/samlinger/oversigt-over-samlinger/samling/dagsordener/dagsorden.aspx?lang=da&day=24-11-2022
 # If I understand this correctly, from the next planned switch to
 # summer time, Greenland will permanently stay at that time, i.e. no
 # switch back to winter time in 2023 will occur.
 #
-# From Paul Eggert (2022-11-25):
-# For now, assume that this applies simultaneously to Ittoqqortoormiit
-# (currently -01) as well as to Nuuk (currently -03), and that
-# Danmarkshavn and Thule are unaffected since they are not using EU
-# rules anyway and some of the remarks were about EU rules.
+# From Paul Eggert (2022-11-28):
+# The official document in Danish
+# https://naalakkersuisut.gl/-/media/naalakkersuisut/filer/kundgoerelser/2022/11/2511/31_da_inatsisartutlov-om-tidens-bestemmelse.pdf?la=da&hash=A33597D8A38CC7038465241119EF34F3
+# says standard time for Greenland is -02, that Naalakkersuisut can lay down
+# rules for DST and can require some areas to use a different time zone,
+# and that this all takes effect 2023-03-25 22:00.  The abovementioned
+# "bekymringer" URL says the intent is no transition March 25, that
+# Greenland will not go back to winter time in fall 2023, and that
+# only America/Nuuk is affected (though further changes may occur).
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Thule	1991	1992	-	Mar	lastSun	2:00	1:00	D
@@ -1135,8 +1146,7 @@ Zone America/Danmarkshavn -1:14:40 -	LMT	1916 Jul 28
 Zone America/Scoresbysund -1:27:52 -	LMT	1916 Jul 28 # Ittoqqortoormiit
 			-2:00	-	-02	1980 Apr  6  2:00
 			-2:00	C-Eur	-02/-01	1981 Mar 29
-			-1:00	EU	-01/+00	2023 Mar 25 24:00
-			 0:00	-	+00
+			-1:00	EU	-01/+00
 Zone America/Nuuk	-3:26:56 -	LMT	1916 Jul 28 # Godthåb
 			-3:00	-	-03	1980 Apr  6  2:00
 			-3:00	EU	-03/-02	2023 Mar 25 22:00
